src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx

```diff
@@ -147,7 +147,7 @@ The log forwarding feature is compatible with the following operating systems:
       </td>
 
       <td>
-        Windows Server 2012, 2016, and 2019, and their service packs.
+        Windows Server 2012, 2016, 2019, 2022 and their service packs.
 
         Windows 10
```

Comment:
@josemore - should we add "and their service packs" after Windows 10 in here? Thanks!

Reply:
I don't think it's needed for Win 10.

Reply:
Thanks!